lib/Federation/BackendNotifier.php

```diff
@@ -101,6 +103,8 @@ public function sendRemoteShare(
 		$protocol['invitedCloudId'] = $invitedCloudId->getId();
 		$protocol['roomName'] = $roomName;
 		$protocol['roomType'] = $roomType;
+		$protocol['roomCallPermissions'] = $roomCallPermissions;
+		$protocol['roomDefaultPermissions'] = $roomDefaultPermissions;
```

Comment:
And description, and call state, and, … 

No this doesn't scale.
And as said, we remove call permissions, please don't add it

Reply:
> And description, and call state, and, …
> 
> No this doesn't scale.

Well, then some other approach is needed. But [the initial state needs to be propagated somehow](https://github.com/nextcloud/spreed/issues/13060).

> And as said, we remove call permissions, please don't add it

This code was written before it was decided that call permissions were going to be removed. Anyway, I will redo it and rebase once call permissions are removed in _main_ and _stable30_.